common/lib/xmodule/xmodule/modulestore/xml_importer.py

```diff
@@ -186,8 +194,10 @@ def __init__(
             default_class='xmodule.raw_module.RawDescriptor',
             load_error_modules=True, static_content_store=None,
             target_id=None, verbose=False,
-            do_import_static=True, create_if_not_present=False,
-            raise_on_failure=False
+            do_import_static=True, do_import_code_lib=True,
+            create_if_not_present=False, raise_on_failure=False,
+            static_content_subdir=DEFAULT_STATIC_CONTENT_SUBDIR,
+            python_lib_filename='python_lib.zip',
```

Comment:
These new args need to be added to the docstring above.

Reply:
Will do